test_util/transaction_test_util.cc

```diff
@@ -39,7 +39,6 @@ RandomTransactionInserter::RandomTransactionInserter(
       cmt_delay_ms_(cmt_delay_ms) {}
 
 RandomTransactionInserter::~RandomTransactionInserter() {
-  last_status_.PermitUncheckedError();
```

Comment:
Since this is a test utility, is there a reason not to uncheck the status in the destructor rather than in other places in the code?

Reply:
swallowing the error in destructor is generally bad practice I think.